Services/CO.CDP.OrganisationInformation.Persistence/DatabaseOrganisationRepository.cs

```diff
@@ -17,6 +17,14 @@ public void Dispose()
             .FirstOrDefaultAsync(t => t.Guid == organisationId);
     }
 
+    public async Task<Organisation?> Find(int organisationId)
```

Comment:
I recommend adding test coverage for the Find.

Reply:
good spot, added.